src/elements/content-uploader/__tests__/ItemRemove.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import noop from 'lodash/noop';
+
+import PlainButton from '../../../components/plain-button';
+import { ItemRemoveBase as ItemRemove } from '../ItemRemove';
+
+import { STATUS_ERROR, STATUS_IN_PROGRESS, STATUS_STAGED } from '../../../constants';
+
+describe('elements/content-uploader/ItemRemove', () => {
+    test.each`
+        status
+        ${STATUS_IN_PROGRESS}
+        ${STATUS_STAGED}
+        ${STATUS_ERROR}
```

Comment:
there's another syntax you can use since this table only has one column

I think possibly this:
`test.each([STATUS_ERROR, STATUS_IN_PROGRESS, STATUS_STAGED])`